Updated `acm-button.svelte` component styling

EthanThatOneKid opened the pull request:
Implemented @mikeploythai's button design. Now, a smoother transition happens when the user hovers over any button element on the site.

Default style:

![image](https://user-images.githubusercontent.com/31261035/116947610-107e7f80-ac32-11eb-850d-62bfb53ed718.png)

Style on hover:

![image](https://user-images.githubusercontent.com/31261035/116947648-2724d680-ac32-11eb-9334-b49f51f60769.png)

EthanThatOneKid committed:
Updated `acm-button.svelte` component styling

diamondburned commented on src/components/utils/acm-button.svelte:
```diff
@@ -12,20 +12,25 @@
+    color: var(--acm-light);
+    background-color: var(--acm-dark);
+    -webkit-filter: drop-shadow(0 2px 10px rgba(23, 26, 28, 0.5));
+    filter: drop-shadow(0 2px 10px rgba(23, 26, 28, 0.5));
```
Does `box-shadow` not work? It should work with `border-radius`.

EthanThatOneKid replied:
`box-shadow` probably works great. I was using the same properties as @mikeploythai's implementation, but `box-shadow` makes more sense here after further consideration. Resolved by commit <https://github.com/EthanThatOneKid/acmcsuf.com/pull/111/commits/e78d5e5f3747ea09896cf4b811a86335598bf6c5>.

EthanThatOneKid committed:
Update acm-button.svelte

Addressed @diamondburned's comment, <https://github.com/EthanThatOneKid/acmcsuf.com/pull/111#discussion_r625442355>.
